convex/imageHttp.ts

```
@@ -0,0 +1,84 @@
+    return createErrorResponse('Internal server error', 500);
+  }
+});
+
+/**
+ * HTTP action to get image URLs from Convex storage
+ */
+export const getImageUrlHttp = httpAction(async (ctx, request) => {
+  // Validate authentication with user verification
+  const authResult = await validateAuthWithUser(request);
+  if (!authResult.success) {
+    return createErrorResponse(authResult.error!, 401);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)